bin/node/src/service.rs

```diff
@@ -5,17 +5,16 @@ use std::{
     sync::Arc,
 };
 
+use fake_runtime_api::fake_runtime::RuntimeApi;
 use finality_aleph::{
     build_network, run_validator_node, AlephBlockImport, AlephConfig, AllBlockMetrics,
     BlockImporter, BuildNetworkOutput, ChannelProvider, Justification, JustificationTranslator,
     MillisecsPerBlock, RateLimiterConfig, RedirectingBlockImport, SessionPeriod,
     SubstrateChainStatus, SyncOracle, TracingBlockImport, ValidatorAddressCache,
 };
 use log::warn;
-use primitives::{
-    fake_runtime_api::fake_runtime::RuntimeApi, AlephSessionApi, Block, DEFAULT_BACKUP_FOLDER,
-    MAX_BLOCK_SIZE,
-};
+use pallet_aleph_runtime_api::AlephSessionApi;
```

Comment:
Do we need that `AlephSessionApi` import there?

Reply:
Actually it is needed, there's below compilation error if it's not there:
```
error[E0599]: no method named `session_period` found for struct `ApiRef<'_, RuntimeApiImpl<Block<Header<u32, BlakeTwo256>, UncheckedExtrinsic>, ...>>` in the current scope
   --> bin/node/src/service.rs:198:14
    |
196 | /         client
197 | |             .runtime_api()
198 | |             .session_period(finalized)
    | |             -^^^^^^^^^^^^^^ method not found in `ApiRef<'_, RuntimeApiImpl<Block<Header<u32, BlakeTwo256>, UncheckedExtrinsic>, ...>>`
    | |_____________|
    | 
    |
    = note: the full type name has been written to '/home/ec2-user/remote-builds/17432567434322764125/target/release/deps/aleph_node-d7a11b894232f461.long-type-17325937729136008227.txt'
    = help: items from traits can only be used if the trait is in scope
help: the following trait is implemented but not in scope; perhaps add a `use` for it:
    |
3   + use pallet_aleph_runtime_api::AlephSessionApi;
```